src/applications/third-party-apps/api/index.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable camelcase */
+
+// Node modules.
+import appendQuery from 'append-query';
+// Relative imports.
+import { apiRequest } from 'platform/utilities/api';
+import { createMockResults, normalizeResponse } from '../helpers';
+
+export const fetchResultsApi = async (options = {}) => {
+  // Derive options properties.
+  const category = options?.category;
+  const platform = options?.platform;
+  const mockRequest = options?.mockRequest;
+  const page = options?.page;
+  const per_page = options?.perPage;
```

Comment:
How come `per_page` uses an underscore? I'm not certain, but I thought camelCase query params would be converted by the API automatically